README.md

```diff
@@ -354,20 +359,75 @@ Submit the case (see CollectionCase.java) by swapping `<EVENT_JSON>` from above
 
 And send And send `python publisher.py local publish event_case-update`
 
+4) **Survey Update**
+
+Submit the case (see SurveyUpdate.java) by swapping `<EVENT_JSON>` from above to the json below (including the tripple quotes)
+
+    """{
```

Comment:
doesn't render right.  Probably need triple single quotes instead of double quotes

Reply:
done a different way